assets/src/application/ui/input/EspressoButton/Tooltip.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import classNames from 'classnames';
+import { isMobile } from 'react-device-detect';
+import { Tooltip as DefaultTooltip } from 'antd';
+import { TooltipPropsWithTitle } from 'antd/lib/tooltip';
+import { __ } from '@wordpress/i18n';
+
+import { isEmpty } from '@appServices/utilities/string';
+
+interface TicketsChainedButtonProps extends TooltipPropsWithTitle {
+	button: JSX.Element;
+	buttonText?: string;
+	tooltip?: string;
+}
+
+const Tooltip: React.FC<TicketsChainedButtonProps> = ({ button, buttonText, tooltip, ...props }) => {
+	const mobileTooltip = isEmpty(buttonText) ? (
+		<div>
+			{button}
+			<div className='esprs-button__tooltip'>{tooltip}</div>
+		</div>
+	) : (
+		button
+	);
+
+	return isMobile ? (
+		mobileTooltip
+	) : (
+		<DefaultTooltip title={tooltip} {...props}>
+			{button}
+		</DefaultTooltip>
+	);
```

Comment:
sorry this ended up being a bit more convoluted than expected...
couldn't the mobile "tooltip" (might be better to call it `mobileHelpText`) just be appended onto the button with `display:none` set for any screen above 480px?

Reply:
updated, but I can't apply css since we need a different HTML structure to display differently on mobile

![image](https://user-images.githubusercontent.com/1477453/77541637-9798ea00-6ead-11ea-850c-6215c5708505.png)